resources/js/frontend/wc-payplug-apple_pay_cart-blocks.js

```diff
@@ -1,19 +1,235 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 import { getSetting } from '@woocommerce/settings';
+import {
+	apple_pay_CancelOrder,
+	apple_pay_getShippings,
+	apple_pay_Payment,
+	apple_pay_PlaceOrderWithDummyData,
+	apple_pay_UpdateOrder
+} from "./helper/wc-payplug-apple_pay-requests";
 const settings = getSetting( 'apple_pay_data', {} );
 
 const ApplePayCart = ( props ) =>{
-	return (
-		<>
-			<div id="apple-pay-button-wrapper">
-				<apple-pay-button
-					buttonstyle="black"
-					type="pay"
-					locale={settings?.payplug_locale}
-				></apple-pay-button>
-			</div>
-		</>
-	)
+
+	let $apple_pay_button = jQuery('apple-pay-button');
+	let session = null;
+	let response = null;
+	const { eventRegistration, emitResponse, shippingData } = props;
+	const { onCheckoutValidation, onPaymentSetup, onCheckoutSuccess, onShippingRateSelectSuccess } = eventRegistration;
+
+	let apple_pay = {
```

Comment:
If the variable will not be changed, should be a const

Reply:
Agreed, I'll change it